Makefile

```diff
@@ -135,6 +143,15 @@ test-integration: manifests generate fmt vet envtest ginkgo ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) --arch=amd64 use $(ENVTEST_K8S_VERSION) -p path)" \
 	$(GINKGO) --junit-report=junit.xml --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET)
 
+USE_EXISTING_CLUSTER ?= true
+.PHONY: test-e2e-kind
+test-e2e-kind: USE_EXISTING_CLUSTER=true
```

Comment:
why not false?

Reply:
Maybe I misunderstood.  

I thought `test-e2e` would be the e2e test in the CI (with creation/deletion of kind) and then test-e2e-kind would be for existing cluster. 

To be honest, both require kind as I am loading the image into kind.  The main difference is whether or not I assume a kind cluster exists.

Reply:
I think the CI should call `test-e2e-kind`.

`test-e2e` can be for an existing cluster (which might or might not be kind)

Reply:
> To be honest, both require kind as I am loading the image into kind

Ah right... Ideally we shouldn't make that assumption. I might want to upload an image to gcr and test on a GKE cluster :)

Reply:
> Ah right... Ideally we shouldn't make that assumption. I might want to upload an image to gcr and test on a GKE cluster :)

My feeling is that we file that as a followup.  It wouldn't be difficult to do but I'd like to get these tests merged so we can get these running on PRs.  

If you don't object, I'll create an issue to make the e2e tests (or another target) to run on an exisiting cluster where the images are stored in a repo somewhere.